### PR TITLE
📄 Improve code reviews docs page

### PR DESCRIPTION
**Pull request by danyf90**
Resolves https://github.com/tuist/tuist/issues/YYY
Request for comments document (if applies):

### Short description 📝

> Describe here the purpose of your PR.

### Checklist ✅

- [ ] The code architecture and patterns are consistent with the rest of the codebase.
- [ ] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
- [ ] The `CHANGELOG.md` has been updated to reflect the changes. In case of a breaking change, it's been flagged as such.
- [ ] In case the PR introduces changes that affect users, the documentation has been updated.

### CODE REVIEW

**Review by fortmarek — APPROVED** [review]

File: projects/docs/docs/contributors/code-reviews.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ must include documentation.
 Pull requests should add an entry to that file describing in one sentence what the change is about.
 They should link to the pull request and mention the author or authors of the changes.
 
-**✅ Tests:** Continuous integration must be happy with the changes.
+**🚦 Continuous integration:** Continuous integration must be happy with the changes.
 The pipelines are designed to bring an extra level of confidence and validate that the changes are right.
 A red CI blocks the merge of the pull request.
+
+**✅ Merging:** A pull request is ready to be merged once there are at least 2 approvals from [Tuist core members](core-team.md).
```
Comment (pepibumur) [inline projects/docs/docs/contributors/code-reviews.md:66]: Note that the path is [/contributors/core-team](https://docs.tuist.io/contributors/core-team)

**Review by pepibumur — APPROVED** [review]

File: projects/docs/docs/guides/dependencies.md [hunk 2]
```diff
@@ -134,4 +134,4 @@ It defines a dependency with a pre-compiled xcframework.
 
 It defines a dependency from an external dependency defined in the `Dependencies.swift` file.
 
-For more information, have a look at the [dedicated section](third-party-dependencies.md).
+For more information, have a look at the [dedicated section](/guides/third-party-dependencies.md).
```
Comment (pepibumur) [inline projects/docs/docs/guides/dependencies.md:137]: I'd remove the `.md` extension

File: projects/docs/docs/contributors/code-reviews.md [hunk 3]
```diff
@@ -59,6 +59,9 @@ must include documentation.
 Pull requests should add an entry to that file describing in one sentence what the change is about.
 They should link to the pull request and mention the author or authors of the changes.
 
-**✅ Tests:** Continuous integration must be happy with the changes.
+**🚦 Continuous integration:** Continuous integration must be happy with the changes.
 The pipelines are designed to bring an extra level of confidence and validate that the changes are right.
 A red CI blocks the merge of the pull request.
+
+**✅ Merging:** A pull request is ready to be merged once there are at least 2 approvals from [Tuist core members](/contributors/core-team.md).
```
Comment (pepibumur) [inline projects/docs/docs/contributors/code-reviews.md:66]: I'd remove the `.md` extension.

Comment (danyf90) [reply projects/docs/docs/contributors/code-reviews.md:66]: I'll fix it in https://github.com/tuist/tuist/pull/3347 to avoid going through CI again :D

**Pull request merged**